integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -662,16 +663,18 @@ object CommonUtil {
     val maxColumnsInt = getMaxColumnValue(maxColumns)
     if (maxColumnsInt != null) {
       if (columnCountInSchema >= maxColumnsInt) {
-        sys.error(s"csv headers should be less than the max columns: $maxColumnsInt")
+        CarbonException.analysisException(
+          s"csv headers should be less than the max columns: $maxColumnsInt")
       } else if (maxColumnsInt > CSVInputFormat.THRESHOLD_MAX_NUMBER_OF_COLUMNS_FOR_PARSING) {
-        sys.error(s"max columns cannot be greater than the threshold value: ${
-          CSVInputFormat.THRESHOLD_MAX_NUMBER_OF_COLUMNS_FOR_PARSING
-        }")
+        CarbonException.analysisException(
+          s"max columns cannot be greater than the threshold value: ${
```

Comment:
move 
```
${
 +            CSVInputFormat.THRESHOLD_MAX_NUMBER_OF_COLUMNS_FOR_PARSING
 +          }"
```
into one line, like line 667. the same for line 677

Reply:
Ok, I have modified it.